assets/app/views/project.html

```diff
@@ -149,9 +149,11 @@ <h2 ng-if="!displayRouteByService[serviceId]">
                             and {{numRemaining}} {{numRemaining == 1 ? "other" : "others"}}
                           </span>
                         </span>
+                        <div class="spacer">&nbsp;</div>
```

Comment:
We've had to handle that in other places by making our OOTB links register
themselves as extensions :-\

On Thu, Apr 14, 2016 at 2:56 PM, Stan Lewis <notifications@github.com>
wrote:

> In assets/app/views/project.html
> <https://github.com/jwforres/origin/pull/35#discussion_r59771913>:
>
> > @@ -149,9 +149,11 @@ <h2 ng-if="!displayRouteByService[serviceId]">
> >                              and {{numRemaining}} {{numRemaining == 1 ? "other" : "others"}}
> >                            </span>
> >                          </span>
> > +                        <div class="spacer">&nbsp;</div>
>
> Totally agree. Ideally I would have made "service-links" below a UL and
> then just changed the 'create route' link (Think it was a span originally)
> accordingly. However, I don't think there's a way currently to have static
> markup within an element that's managed by the extension-point directive,
> i.e. extensions will overwrite anything I put in there.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jwforres/origin/pull/35/files/d7af134f8979319f461b5ad68aa9e9377f0b2fd0#r59771913>
>